src/main/webapp/partials/instructorHelpArchiving.jsp

```diff
@@ -0,0 +1,136 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<h4>
+    <a name="archiving">Archiving</a>
+</h4>
+<div id="contentHolder">
+    <br>
+    <ol style="list-style-type: none;">
+        <li>
+            <span class="text-bold">
+                <a name="archivingCourse">
+                    <h3>1. Archiving a course</h3>
+                </a>
+            </span>
+            <div>
+                You can archive a course so that it doesn't appear in your home page anymore. You cannot create feedback sessions, or enroll students, or edit an archived course.
+                <br>Go to your
+                <b>'Home'</b> page. You will see a separate panel for each course and a table of feedback sessions inside it. It looks something similar to this:
```

Comment:
To be consistent with the tense used in the previous instances, `looks` -> `should look`.